analysis/pf_dataset.py

```diff
@@ -33,102 +38,117 @@ def get_pf_clinical_events(pharmacy_first_ids):
 # - chronic sinusitis and immunosuppressed individuals for acute sinusitis
 def get_uncomplicated_uti_denominator(index_date, selected_events, pregnancy_codelist):
```

Comment:
I realised that we should add patients to these functions as well, so that all we provide all information needed to perform the calculations in this functions are provided (see also comment below):

```suggestion
def get_uncomplicated_uti_denominator(index_date, patients, selected_events, pregnancy_codelist):
```